advocacy_docs/edb-postgres-ai/console/estate/reference/metrics_ingestion.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: Metrics Ingestion into the EDB Postgres AI Estate
+navTitle: Metrics Ingestion
+description: The metrics that can be ingested into the EDB Postgres AI Estate.
+---
+
+All agents in the EDB Postgres AI Estate can send metrics to the Estate control plane.
+
+Currently the following metrics are ingested into the Estate control plane:
```

Comment:
This list applies to bare metal instances only... although you did say 'all agents' so that's kind of implied. Although actually the bare metal agent also sends some data about the host OS although it's not exposed anywhere in UI.